spec/dummy/spec/models/secure_meeting_spec.rb

```diff
@@ -0,0 +1,18 @@
+require_relative '../../spec/factories/secure_meeting'
+
+describe SecureMeeting  do
+  context "A model with a UUID as a primary key" do
+    before(:all) do
+    SecureMeeting.delete_all
+    end
+
+    it "updates the relevant record" do
+      meeting_uuid = FactoryGirl.create(:secure_meeting)
```

Comment:
variable naming here:  I'd expect a variable named UUID to return a UUID, not an object that has a UUID.

I'd probably just call this `meeting`